sql/updates/world/master/2023_mm_dd_xx_world.sql

```diff
@@ -0,0 +1,48 @@
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 27 AND `SourceEntry` IN (SELECT `ID` FROM `graveyard_zone`);
+INSERT INTO
+    `conditions` (
+        `SourceTypeOrReferenceId`,
+        `SourceEntry`,
+        `ConditionTypeOrReference`,
+        `ConditionValue1`,
+        `Comment`
+    )
+SELECT
+    27,
+    `ID`,
+    6,
+    `Faction`,
+    CONCAT(
+        'Graveyard - ',
+        `ID`,
+        ' - Team ',
+        CASE
+            WHEN `Faction` = 469 THEN 'Alliance'
+            WHEN `Faction` = 67 THEN 'Horde'
+            ELSE `Faction`
+        END
+    )
+FROM `graveyard_zone`
+WHERE `Faction` <> 0
+GROUP BY 1, 2, 3, 4, 5;
```

Comment:
what is your intention with this group by?

Reply:
toi avoid duplicates. But this will be removed later when support for ghostzone in conditions are added